src/ert/_c_wrappers/analysis/analysis_module.py

```diff
@@ -1,162 +1,216 @@
+DEFAULT_IES_MIN_STEPLENGTH = 0.30
+DEFAULT_IES_DEC_STEPLENGTH = 2.50
+DEFAULT_TRUNCATION = 0.98
+DEFAULT_INVERSION = 0
+
+
+class AnalysisMode(str, Enum):
+    ITERATED_ENSEMBLE_SMOOTHER = "IES_ENKF"
+    ENSEMBLE_SMOOTHER = "STD_ENKF"
+
+
+def get_mode_variables(module: AnalysisMode) -> Dict[str, "VariableInfo"]:
+    es_variables: Dict[str, "VariableInfo"] = {
+        "IES_INVERSION": {
+            "type": int,
+            "min": 0,
+            "value": DEFAULT_INVERSION,
+            "max": 3,
+            "step": 1,
+            "labelname": "Inversion algorithm",
+        },
+        "ENKF_TRUNCATION": {
+            "type": float,
+            "min": -2.0,
+            "value": DEFAULT_TRUNCATION,
+            "max": 1,
+            "step": 0.01,
+            "labelname": "Singular value truncation",
+        },
+    }
+    ies_variables = {
         "IES_MAX_STEPLENGTH": {
             "type": float,
             "min": 0.1,
+            "value": DEFAULT_IES_MAX_STEPLENGTH,
             "max": 1.00,
             "step": 0.1,
             "labelname": "Gauss–Newton maximum steplength",
         },
         "IES_MIN_STEPLENGTH": {
             "type": float,
             "min": 0.1,
+            "value": DEFAULT_IES_MIN_STEPLENGTH,
             "max": 1.00,
             "step": 0.1,
             "labelname": "Gauss–Newton minimum steplength",
         },
         "IES_DEC_STEPLENGTH": {
             "type": float,
             "min": 1.1,
+            "value": DEFAULT_IES_DEC_STEPLENGTH,
             "max": 10.00,
             "step": 0.1,
             "labelname": "Gauss–Newton steplength decline",
         },
-        "IES_INVERSION": {
-            "type": int,
-            "min": 0,
-            "max": 3,
-            "step": 1,
-            "labelname": "Inversion algorithm",
-        },
-        "ENKF_TRUNCATION": {
-            "type": float,
-            "min": -2.0,
-            "max": 1,
-            "step": 0.01,
-            "labelname": "Singular value truncation",
-        },
+        **es_variables,
     }
-
-    def __init__(self, type_id):
-        c_ptr = self._alloc(type_id)
-        if not c_ptr:
-            raise KeyError(f"Failed to load internal module:{type_id}")
-
-        super().__init__(c_ptr)
-
-    def getVariableNames(self) -> List["VariableName"]:
-        items = []
-        for name in AnalysisModule.VARIABLE_NAMES:
-            if self.hasVar(name):
-                items.append(name)
-        return items
-
-    def getVariableValue(self, name: "VariableName") -> Union[int, float, bool]:
-        self.__assertVar(name)
-        variable_type = self.getVariableType(name)
-        if variable_type == float:
-            return self.getDouble(name)
-        elif variable_type == bool:
-            return self.getBool(name)
-        elif variable_type == int:
-            return self.getInt(name)
+    if module == AnalysisMode.ENSEMBLE_SMOOTHER:
+        return es_variables
+    return ies_variables
```

Comment:
Is the typing complaint fixed? I am asking because I don't get any automatic typing complaint from my IDE and the typing tests are passing

Reply:
it's a bit inconvenient, because you have to manually look for them - they show up as inline comments in the changes tab in the PR. it's gone in the new PR!